syndeo-template/src/main/java/com/google/cloud/syndeo/common/ProviderUtil.java

```diff
@@ -136,6 +133,17 @@ public static PCollectionRowTuple applyConfigs(
       }
 
       tuple = tuple.apply(spec.inputId, transform.buildTransform());
+
+      if (tuple.getAll().containsKey(ERRORS_TAG)) {
+        // TODO(pabloem): Deal with errors.
+        PCollectionRowTuple.of("input", tuple.get(ERRORS_TAG))
```

Comment:
so right now we are just incrementing the metric to detect the error right.

Reply:
that's correct Manav! We can figure out how to act on that afterwards